packages/nx-plugin/src/generators/configuration/__snapshots__/generator.integration.test.ts.snap

```diff
@@ -13,3 +13,17 @@ const config: CoreConfig = {
 export default config;
 "
```

Comment:
The previous snapshot is obsolete and should be deleted:

![image](https://github.com/user-attachments/assets/8bffd425-f0b7-4684-9ada-0b1931194691)